pkg/curatedpackages/package.go

```diff
@@ -0,0 +1,38 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+const (
+	minWidth = 16
+	tabWidth = 8
+	padding  = 0
+	padChar  = '\t'
+	flags    = 0
+)
+
+func DisplayPackages(packages []api.BundlePackage) {
+	w := new(tabwriter.Writer)
+	w.Init(os.Stdout, minWidth, tabWidth, padding, padChar, flags)
+	defer w.Flush()
```

Comment:
nit: it's probably a bit cleaner to add this just after creating the writer in line 22